docs/helpers/snippets/decodeJwt.vue

```diff
@@ -0,0 +1,22 @@
+<script setup lang="ts">
+import type { CredentialResponse } from "vue3-google-signin";
+import { GoogleSignIn } from "vue3-google-signin";
```

Comment:
Do we need this import?

Reply:
I actually wasn't sure as I use autoimport in all my vue projects this far.
If you believe it is not, I'll remove it.